src/main/java/com/google/growpod/servlets/UserServlet.java

```diff
@@ -65,6 +65,17 @@
     return Collections.unmodifiableMap(map);
   }
 
+  private static final Map<String, List<String>> USER_GARDEN_ADMIN_LIST_MAP =
+      createUserGardenAdminListMap();
+
+  private static Map<String, List<String>> createUserGardenAdminListMap() {
+    Map<String, List<String>> map = new HashMap<String, List<String>>();
```

Comment:
Just to clarify, this was the list of gardens a user administers from PR#4 ?

Reply:
Yes